Update documentation to 0.902 (#11)
Neogoma/stardust-SDK-doc: developer/api/0-intro.md

# Intro

> First of all in order to use the API endpoints you need to use a paid plan. If you didn't subscribe a paid plan yet, you can do it [here](https://stardust.neogoma.com/pricing).

Reviewer: ```suggestion
First of all, in order to use the API endpoints, you need to use a paid plan. If you haven't subscribed to a paid plan yet, you can do it [here](https://stardust.neogoma.com/pricing).
```

## Headers

> The follwing headers need to be setup to authenticate your account on the request. You can find your token [here](https://stardust.neogoma.com/profile).

Reviewer: ```suggestion
The following headers need to be setup to authenticate your account in the request. You can find your API token [here](https://stardust.neogoma.com/profile).
```

| Name | Value |
| --- | --- |
| Authorization | "Bearer "+[API KEY] |

> If you don't provide the token or you try to access data that are not yours you trigger a 403 forbidden error.

Reviewer: ```suggestion
If you don't provide the token or you try to access data that are not yours, a HTTP 403 forbidden error will be returned in the response.
```

Reviewer 2: ```suggestion
If you don't provide the token or you try to access data that is not yours, a HTTP 403 forbidden error will be returned in the response.
```

Reviewer 2: change to "is" instead of "are"
